gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -793,6 +910,73 @@ public static LicenseOption fields(LicenseField... fields) {
   }
 
   /**
+   * Class for specifying operation get options.
+   */
+  class OperationOption extends Option {
+
+    private static final long serialVersionUID = -4572636917684779912L;
+
+    private OperationOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the operation's fields to be returned by the RPC call. If this
+     * option is not provided all operation's fields are returned. {@code OperationOption.fields}
+     * can be used to specify only the fields of interest. {@link Operation#operationId()} is
+     * always returned, even if not specified.
+     */
+    public static OperationOption fields(OperationField... fields) {
+      return new OperationOption(ComputeRpc.Option.FIELDS, OperationField.selector(fields));
+    }
+  }
+
+  /**
+   * Class for specifying operation list options.
+   */
+  class OperationListOption extends Option {
+
+    private static final long serialVersionUID = -1509532420587265823L;
+
+    private OperationListOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify a filter to the operations being listed.
+     */
+    public static OperationListOption filter(OperationFilter filter) {
+      return new OperationListOption(ComputeRpc.Option.FILTER, filter.toPb());
+    }
+
+    /**
+     * Returns an option to specify the maximum number of operations to be returned.
```

Comment:
Interesting, if the behavior is the same (max results per page rather than max results overall), I feel like the cloud docs are slightly inconsistent. (While those docs say "per page", storage's and bigquery's list docs don't say "per page".)  I guess all those list functions return a page token to get more results?

I used pageSize in Resource Manager rather than maxResults (in part because the REST API called it pageSize rather than maxResults).  I prefer page size over max results, but at the very least we should standardize terminology in our library, so I'll create an issue.